apps/camera/js/controllers/camera.js

```diff
@@ -79,10 +79,10 @@ CameraController.prototype.bindEvents = function() {
   app.on('activity:pick', this.onPickActivity);
   app.on('keydown:capture', this.onCaptureKey);
   app.on('keydown:focus', this.onFocusKey);
-  app.on('timer:ended', this.capture);
+  app.on('hidden', this.shutdownCamera);
+  app.on('click', this.clearCountdown);
```

Comment:
The main change of this patch is that any click event on the camera will now cancel the countdown. Am I right? For next time, It would be useful to provide a high level description on the commit message of what the problem is and the followed approach to fix it.